client/jsonrpc_config.go

```diff
@@ -0,0 +1,65 @@
+package client
+
+import (
+	"net/http"
+	"strings"
+	"time"
+)
+
+type EmptyUrlError struct {
```

Comment:
if theres no fields lets convert this to a var
```suggestion
var EmptyUrlError = errors.New("Empty port and IP provided")
```